lib/spack/docs/build_systems/makefilepackage.rst

```diff
@@ -5,9 +5,9 @@
 
 .. _makefilepackage:
 
----------------
-MakefilePackage
----------------
+--------
+Makefile
```

Comment:
Why change the filename here but not the other build systems?

Reply:
That is a detail that slipped though in this huge PR. I'd change the title also in other places if you agree. The rationale is that `Makefile` is the build-system and `MakefilePackage` is the base class for `Packages` with directives, URLs etc.

Reply:
Makes sense to me!